### ISSUE TITLE
--me flag not working

### ISSUE THREAD

**Issue by facundofc** [martindstone/pagerduty-cli#6]
Hello there! I've noticed that the `--me` option to the `incident:list` doesn't work, giving me the following error:

```
Error: Request to /users/me failed: Legacy API tokens aren't supported for this operation
```

I tried with `rest:get --endpoint users/me`, and it works perfectly returning my own user.

I've poken trough the code and I see there's a function `isBearerToken()` which checks if the token is 64 characters long. Mine is 20, which according to the code it renders it a legacy token.

What's the reason for the need of the `isBearerToken()`? As far as PD goes, it does seem an artificial enforcement, given that the API endpoint actually works. I tried digging into the code history but that function was introduced in the initial commit.

I'd be willing to push an MR with the changes necessary, but I wanted first to double check this.

Cheers!

**Comment by martindstone** [comment]
Hi Facundo! So /users/me does work for bearer tokens obtained through pd oauth, and doesn’t work for what I call legacy domain tokens (provisioned by administrators under API Access...) but now it occurs to me that there’s a third type of token that’s provisioned in your individual PD User settings, that looks like a legacy token but may work with /users/me (although the Authorization: header gets constructed differently...) I can check this tomorrow and push a fix for you... can you confirm what kind of token you are using?

Best,
Martin

**Comment by facundofc** [comment]
Hi Martin! Thanks for the speedy reply.

Indeed the token I got from my user settings, much like what's described
here
<https://support.pagerduty.com/docs/generating-api-keys#section-generating-a-personal-rest-api-key>.
Let me know if this is enough to determine the token's type.

Cheers.

**Comment by martindstone** [comment]
Hi Facundo, this should be fixed with 0.0.43, just published. If you're using a user legacy API token, the `--me` option should now work with all the `incident:*` commands. Please reopen if you see anything bad... Thanks!

**Comment by facundofc** [comment]
Works perfect! Thanks!
